docs/faq.md

```diff
@@ -155,10 +155,10 @@
 !!!tip ""
     This is an issue primarily affecting macOS users, but may occur on other devices as well. If you are able to launch the homebrew menu, but you are not seeing some or any of your homebrew, you will need to unset the archive bit with Hekate.
 
-    1. Send the Hekate payload to your Switch by following these instructions: [here](https://nh-server.github.io/switch-guide/user_guide/emummc/sending_payload/)
+    1. Send the Hekate payload to your Switch by following these instructions: [here](https://nh-server.github.io/switch-guide/user_guide/emuMMC/sending_payload/)
```

Comment:
Change the link from https://nh-server.github.io/switch-guide/user_guide/emuMMC/sending_payload/ to "user_guide/emummc/sending_payload.md"